modules/primer-layout/README.md

```diff
@@ -115,7 +115,7 @@ In practice, your columns will look like the example below.
 
 ##### Centered
 
-Columns can be [centered](/utilities/#centering-content) by adding `.centered` to the `.column` class.
```

Comment:
I removed this link since it seemed like we don't have a section about centering content anymore? cc @broccolini